hkdf/src/hkdf.rs

```diff
@@ -60,9 +110,9 @@ where
     D::BlockSize: ArrayLength<u8>,
     D::OutputSize: ArrayLength<u8>,
 {
-    /// Convenience method for [`extract`] when the generated pseudorandom
-    /// key can be ignored and only HKDF-Expand operation is needed. This is
-    /// the most common constructor.
+    /// Convenience method for [`extract`](struct.Hkdf.html#method.extract) when the generated
```

Comment:
Instead of using an explicit link, it's better to use [intra-rusdoc](https://github.com/rust-lang/rust/issues/43466) links. In this case it should look like ``[`extract`][Hkdf::extract]``. Same for the doc at line 145.